kong/types.go

```diff
@@ -10,56 +10,55 @@ import (
-	PreserveHost  *bool               `json:"preserve_host,omitempty" yaml:"preserve_host,omitempty"`
-	Protocols     []*string           `json:"protocols,omitempty" yaml:"protocols,omitempty"`
-	RegexPriority *int                `json:"regex_priority,omitempty" yaml:"regex_priority,omitempty"`
-	Service       *Service            `json:"service,omitempty" yaml:"service,omitempty"`
-	StripPath     *bool               `json:"strip_path,omitempty" yaml:"strip_path,omitempty"`
-	UpdatedAt     *int                `json:"updated_at,omitempty" yaml:"updated_at,omitempty"`
-	SNIs          []*string           `json:"snis,omitempty" yaml:"snis,omitempty"`
-	Sources       []*CIDRPort         `json:"sources,omitempty" yaml:"sources,omitempty"`
-	Destinations  []*CIDRPort         `json:"destinations,omitempty" yaml:"destinations,omitempty"`
-	Tags          []*string           `json:"tags,omitempty" yaml:"tags,omitempty"`
-
-	HTTPSRedirectStatusCode *int `json:"https_redirect_status_code,omitempty" yaml:"https_redirect_status_code,omitempty"`
+	CreatedAt               *float64            `json:"created_at,omitempty" yaml:"created_at,omitempty"`
+	Hosts                   []*string           `json:"hosts,omitempty" yaml:"hosts,omitempty"`
+	Headers                 map[string][]string `json:"headers,omitempty" yaml:"headers,omitempty"`
+	ID                      *string             `json:"id,omitempty" yaml:"id,omitempty"`
+	Name                    *string             `json:"name,omitempty" yaml:"name,omitempty"`
+	Methods                 []*string           `json:"methods,omitempty" yaml:"methods,omitempty"`
+	Paths                   []*string           `json:"paths,omitempty" yaml:"paths,omitempty"`
+	PathHandling            *string             `json:"path_handling,omitempty" yaml:"path_handling,omitempty"`
+	PreserveHost            *bool               `json:"preserve_host,omitempty" yaml:"preserve_host,omitempty"`
+	Protocols               []*string           `json:"protocols,omitempty" yaml:"protocols,omitempty"`
+	RegexPriority           *int64              `json:"regex_priority,omitempty" yaml:"regex_priority,omitempty"`
+	Service                 *Service            `json:"service,omitempty" yaml:"service,omitempty"`
+	StripPath               *bool               `json:"strip_path,omitempty" yaml:"strip_path,omitempty"`
+	UpdatedAt               *float64            `json:"updated_at,omitempty" yaml:"updated_at,omitempty"`
+	SNIs                    []*string           `json:"snis,omitempty" yaml:"snis,omitempty"`
+	Sources                 []*CIDRPort         `json:"sources,omitempty" yaml:"sources,omitempty"`
+	Destinations            []*CIDRPort         `json:"destinations,omitempty" yaml:"destinations,omitempty"`
+	Tags                    []*string           `json:"tags,omitempty" yaml:"tags,omitempty"`
+	HTTPSRedirectStatusCode *int64              `json:"https_redirect_status_code,omitempty" yaml:"https_redirect_status_code,omitempty"`
```

Comment:
@mflendrich, this line is the reason why I changed lll parameters.
I can't find a way to resize it beneath 121 characters with int it's fine but with int64, it isn't

Reply:
golangci supports per-file exemptions, which would be appropriate here. go-kong doesn't currently have any, but deck does: https://github.com/Kong/kong/blob/master/kong.conf.default#L98-L126 is an example for this rule.

Reply:
Nope. Clipboard hurray. https://github.com/Kong/deck/blob/6fd816b2bb4603dc2479beeff8b6e1e7a6e1e167/.golangci.yml#L37-L40 was what I meant.